tests/console/yast2_samba.pm

```diff
@@ -0,0 +1,279 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+    select_console 'root-console';
+
+    # check network at first
+    assert_script_run("if ! systemctl -q is-active network; then systemctl -q start network; fi");
```

Comment:
I am not sure that network is always is working, there is no network test before it or in extratest. This checks it again, the test should stop here if no networking is available.